### PR TITLE
Temp dev

### PR DESCRIPTION
**Pull request by preethasri**
added css for functionality 
removed some bucks

### CODE REVIEW

File: src/Components/Navbar/Navbar.js [hunk 1]
```diff
@@ -25,23 +26,27 @@ const Navbar=({filtersState,filterDispatch})=>{
...
-                      </div>
-                  </div>
+                        <button className="search-btn">
+                           <span className='material-icons-outlined'>search</span>
+                          
+                        </button>
+                         
+                         
+                     
+                        </div>
```
Comment (ranitasahaa1230) [inline src/Components/Navbar/Navbar.js:45]: Run prettier over here

File: src/Context/WatchLaterContext.js [hunk 2]
```diff
@@ -35,28 +35,27 @@ const WatchLaterProvider=({children})=>{
...
-           console.log(err)
-       }
-   }
+    const addToWatchLaterVideos = async video => {
+        try {
+          const response = await addWatchLater(token, video);
+          setWatchLaterVideos(response.data.watchlater);
+          toast.success("Video added to Watch Later");
+        } catch (error) {
+          console.error("ERROR", error);
```
Comment (ranitasahaa1230) [inline src/Context/WatchLaterContext.js:44]: Remove the console,

**Review by ranitasahaa1230 — APPROVED** [review]
Good use of contexts and reducers
Remove the console if not need
Clean UI
Add cursor: pointer to the likes, watch later, delete buttons

File: src/Context/WatchLaterContext.js (hunk 2 above)
Comment (preethasri) [reply src/Context/WatchLaterContext.js:44]: sure 👍

**Pull request merged**